---
layout: default
title: Research
---
### Research
Their research interests include investigating disease related mutations and how they effect the ways that proteins fold and bind their ligands. They are also interested in the timescale problem associated with MD simulation. Being a member of a lab that is part of the [Folding@Home](https://foldingathome.org/) consortium, they have used massively parallel computing to investigate protein folding. To this end, they are also interested in using GPU-accelerated metadynamics simulations to accurately investigate folding and binding to more efficiently predict experimental observables. Information about some of their work can be found below.

### Hemagglutanin(https://vav20.fah.temple.edu/pub/hemagglutanin/index.html)
I am using investigate the binding of miniproteins *de novo* designed by the Baker lab that target the conserved stem region of H1N1 Hemagglutinin (HA2). These miniproteins were selected using a process of affinity maturation, in which all residues were exhaustively mutated to discover potent binders. We aim to assess the ability of MD simulation to model this protein binding reaction and the mechanism by which mutations enhance affinity to HA2. A combination of unbiased and biased simulations were used to build multi-ensemble MSMs of binding for wild type and affinity-matured variants., To test whether MD can predict the experimentally observed affinity-maturation, we used massively parallel expanded ensemble simulations to perform alchemical free energy perturbation (FEP) site-scanning mutagenesis (SSM) of the wild-type HA2 miniprotein binders. These predictions are then used to infer how binding kinetics and pathways are perturbed by mutations.

### [FOX01](https://pubs.acs.org/doi/abs/10.1021/acs.biochem.2c00224)
FOXO1, a member of the family of winged-helix motif Forkhead box (FOX) transcription factors, is the most abundantly expressed FOXO member in mature B cells. Sequencing of diffuse large B-cell lymphoma (DLBCL) tumors and cell lines identified specific mutations in the forkhead domain linked to loss of function. Differential scanning calorimetry and thermal shift assays were used to characterize how eight of these mutations affect the stability of the FOX domain. Mutations L183P and L183R were found to be particularly destabilizing. Electrophoresis mobility shift assays show these same mutations also disrupt FOXO1 binding to their canonical DNA sequences, suggesting that the loss of function is due to destabilization of the folded structure. Computational modeling of the effect of mutations on FOXO1 folding was performed using alchemical free energy perturbation (FEP), and a Markov model of the entire folding reaction was constructed from massively parallel molecular simulations, which predicts folding pathways involving the late folding of helix α3. Although FEP can qualitatively predict the destabilization from L183 mutations, we find that a simple hydrophobic transfer model, combined with estimates of unfolded-state solvent-accessible surface areas from molecular simulations, is able to more accurately predict changes in folding free energies due to mutations. These results suggest that the atomic detail provided by simulations is important for the accurate prediction of mutational effects on folding stability. Corresponding disease-associated mutations in other FOX family members support further experimental and computational studies of the folding mechanism of FOX domains.

### Cyclic Peptides
Peptides and peptidomimetic molecules have promising potential as therapeutics. Being composed of amino acids, these molecules can mimic protein-protein interactions, typically better than small  molecules can, and have a low potential for toxicity. The structures in this project are mimics of “omega loops”, such as portions of stonin-2, a protein that binds Eps15. Eps-15-stonin-2 PPI is involved in the clathrin-mediated endocytosis process, recruiting adapter complex AP-2 to clathrin-coated pits. Disruption of this interaction has displayed reduced infectivity of Ebola, revealing the potential of modulating this pathway as a means of preventative therapy.

There are two main aims of this project. Firstly, we are aiming to explore how these omega loop peptides configure themselves in solution. Secondly, we aim to tackle questions surrounding the preconfiguration of these molecules and their binding to their target. Exploring these questions will provide great insight into the design of protein mimic therapeutics that will allow for more informed design of PPI pathway modulators. These simulations will also present future opportunities for testing tools related to protein-ligand systems that could streamline the peptide-therapeutic design process.
